src/extra/imgui.cpp

```diff
@@ -541,10 +538,13 @@ struct Window : public Base {
   chainblocks::BlocksVar _blks{};
   std::string _title;
   bool firstActivation{true};
-  Var _pos{}, _width{1.0}, _height{1.0};
```

Comment:
why?
I mean we still use relative 0.0, 1.0 I hope even if you added None

Reply:
Yes we do (if the parameter is a float). It is just to support the auto-size-to-content when it is None.